tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -30,4 +29,6 @@ def test_add_capacity(self):
             selector='app=rook-ceph-osd',
             resource_count=result * 3
         )
-        self.ceph_cluster.cluster_health_check(timeout=1200)
+        assert ceph_health_check(
+            namespace=config.ENV_DATA['cluster_namespace'], tries=80
+        ), f"Ceph health is not OK"
```

Comment:
also taking care of in the function call:
```
raise CephHealthException(
            f"Ceph cluster health is not OK. Health: {health}"
        )
```

Reply:
ack